SyncDocs/SyncDocs.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.4.33205.214
```

Comment:
I'm thinking we should add the tool in our main datahub project for now - the idea is to keep this repo for the documentation only